tutorials/other/ssh_keys.md

```diff
@@ -0,0 +1,91 @@
+Copyright 2015 [Nicky Semenza](http://nickysemenza.com)
```

Comment:
I think it's great to have an author field or something so that we can get credit, but I don't think we want everybody claiming copyright to these guides. One of the goals of this course is to be forked by other universities and used there.

@leviathan747 any thoughts? How does the GPL license interact with copyrights on individual files? I really don't know  enough about GPL/copyright/copyleft